x/funders/keeper/logic_funders.go

```diff
@@ -167,9 +195,9 @@ func (k Keeper) ensureFreeSlot(ctx sdk.Context, newFunding *types.Funding, fundi
 		return nil
 	}
 
-	params := k.GetParams(ctx)
+	whitelist := k.GetCoinWhitelistMap(ctx)
```

Comment:
whitelist is retrieved here, although it's first use is at line 211. Maybe move declaration